randovania/games/super_metroid/logic_database/Maridia.txt

```diff
@@ -661,7 +661,7 @@ Extra - asset_id: 145
           # Shinespark
           Gravity Suit and Speed Booster and Shinespark (Beginner) and Shinespark Damage ≥ 100
           # Snail Climb and Clip: https://youtu.be/ud0I-bb6mVw
-          Movement (Expert) and Snail Climb (Intermediate)
+          Knowledge (Expert) and Snail Climb (Intermediate)
```

Comment:
you should bump this down to knowledge intermediate or advanced. expert is definitely too high.

![grafik](https://github.com/randovania/randovania/assets/38186597/72994a59-6852-4564-8fcf-0501d6d039dc)

Reply:
lowered